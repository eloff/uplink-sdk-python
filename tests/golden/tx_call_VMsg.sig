r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 51632070471693333503214576369904991998198604773897170650995389500896993293730
