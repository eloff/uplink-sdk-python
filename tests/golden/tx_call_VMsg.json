{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "contents": "Hello world", 
                        "tag": "VMsg"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE01MTYzMjA3MDQ3MTY5MzMzMzUwMzIxNDU3NjM2OTkwNDk5MTk5ODE5ODYwNDc3Mzg5NzE3MDY1MDk5NTM4OTUwMDg5Njk5MzI5MzczMA=="
}